src/Domain/CoordinateMaps/EquatorialCompression.cpp

```diff
@@ -67,41 +77,47 @@ EquatorialCompression::angular_distortion_jacobian(
   const ReturnType& z = coords[2];
   const ReturnType radius = sqrt(square(x) + square(y) + square(z));
   const ReturnType rho =
-      sqrt(square(x) + square(y) + square(inverse_alpha * z));
-  // Various common factors:
-  const ReturnType lambda1 = (pow<2>(inverse_alpha) - 1.0) * z * z;
-  const ReturnType lambda2 = (1.0 - pow<2>(inverse_alpha)) * (x * x + y * y);
-  const ReturnType lambda3 = pow<2>(rho * radius);
-  const ReturnType lambda4 = radius * pow<3>(rho);
+      sqrt(square((index_pole_axis_ == 0) ? inverse_alpha * x : x) +
+           square((index_pole_axis_ == 1) ? inverse_alpha * y : y) +
+           square((index_pole_axis_ == 2) ? inverse_alpha * z : z));
 
   auto jacobian_matrix =
       make_with_value<tnsr::Ij<tt::remove_cvref_wrap_t<T>, 3, Frame::NoFrame>>(
           dereference_wrapper(coords[0]), 0.0);
 
-  get<0, 0>(jacobian_matrix) = (x * x * lambda1 + lambda3);
-  get<1, 0>(jacobian_matrix) = x * y * lambda1;
-  get<2, 0>(jacobian_matrix) = inverse_alpha * x * z * lambda1;
-
-  get<0, 1>(jacobian_matrix) = get<1, 0>(jacobian_matrix);
-  get<1, 1>(jacobian_matrix) = (y * y * lambda1 + lambda3);
-  get<2, 1>(jacobian_matrix) = inverse_alpha * y * z * lambda1;
-
-  get<0, 2>(jacobian_matrix) = x * z * lambda2;
-  get<1, 2>(jacobian_matrix) = y * z * lambda2;
-  get<2, 2>(jacobian_matrix) = inverse_alpha * (z * z * lambda2 + lambda3);
+  for (size_t i = 0; i < 3; i++) {
+    for (size_t j = 0; j < 3; j++) {
+      const size_t k = (index_pole_axis_ + j) % 3;
+      jacobian_matrix.get(i, k) =
+          gsl::at(coords,i) * gsl::at(coords,k) * (pow<2>(inverse_alpha) - 1.0);
+      if (j == 0) {
+        jacobian_matrix.get(i, k) *=
+            (square(gsl::at(coords,index_pole_axis_)) - square(radius));
```

Comment:
I missed this before, but why do you have both `square` and `pow<2>` here (and below)? I'd say pick one and use that throughout. I think SpECTRE convention is to use `square` and `cube` from `ConstantExpressions.hpp`.